classroom/snippets/src/main/java/ReturnStudentSubmission.java

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// [START classroom_return_student_submissions_class]
+
+import com.google.api.client.googleapis.json.GoogleJsonError;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.classroom.Classroom;
+import com.google.api.services.classroom.ClassroomScopes;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import java.util.Collections;
+
+/* Class to demonstrate the use of Classroom Return StudentSubmissions API. */
+public class ReturnStudentSubmission {
+  /**
+   * Return student submission.
+   *
+   * @param courseId - identifier of the course.
+   * @param courseWorkId - identifier of the course work.
+   * @param id - identifier of the student submission.
```

Comment:
Perhaps this should be `userId` or `studentId` to make it clearly distinct from the other identifiers?

Reply:
Ah, gotcha. This happens a few times in the API, where the generic `id` field could really benefit from a more descriptive name. E.g., `id` is returned by the add-ons API for an attachment ID upon creation, but it's called `attachmentId` in other contexts.